prod/westeurope/notifications/functions_pushnotif/function_app/terragrunt.hcl

```diff
@@ -90,6 +99,16 @@ inputs = {
 
     APPINSIGHTS_SAMPLING_PERCENTAGE = 5
 
+    # ------------------------------------
+    # Variable used during transition to new NH management
+
+    # Possible values : "none" | "all" | "beta" | "canary"
+    NH_PARTITION_FEATURE_FLAG             = "none"
+    BETA_USERS_STORAGE_CONNECTION_STRING  = dependency.storage_test-users.outputs.primary_connection_string
+    BETA_USERS_TABLE_NAME                 = dependency.storage_test-users_table_nh-test-users.outputs.name
+    CANARY_USERS_REGEX                    = "^([(0-9)|(a-f)|(A-F)]{63}0)|([(0-9)|(a-f)|(A-F)]{62}[(0-7)]{1}1)$"
```

Comment:
This `[(0-9)|(a-f)|(A-F)]` is basically `*` in a hexadecimal world
Anyway, as regex are not so talkative, think about add a comment about the regex intention.

Reply:
Also, to match last character: https://stackoverflow.com/questions/45841090/regex-to-match-last-character-in-string

Reply:
With this regex we're also checking that the input is a right 64 hex characters string, just to be sure. Dont' know if it's necssary, but it's more robust

Reply:
Added comment in [810e84b](https://github.com/pagopa/io-infrastructure-live-new/pull/498/commits/810e84b3f93113aa16a36fd2b2fb75086bbab911)